agents/Group23/MCTSAgent.py

```diff
@@ -46,6 +47,7 @@ def __init__(self, colour: Colour, max_simulation_length: float = 2.5):
         super().__init__(colour)
         self.max_simulation_length = max_simulation_length # max length of a simulation
         self.root = None
+        self.zobrist_hasher = ZobristHasher(11)
```

Comment:
They very likely won't change this, but should we set the same board size as board incase they use a different board in the assessment?

Reply:
We could do but I figured I'd hard code the 11 since all of the openings that I encoded into the transposition table will only work for an 11x11 table, so the raw number just saves an attribute lookup and a tiny fraction of a second.

Reply:
Ah I see, that makes sense :)